pytorch_lightning/trainer/trainer.py

```diff
@@ -1168,6 +1171,18 @@ def call_hook(self, hook_name: str, *args, **kwargs) -> Any:
 
         return output
 
+    def _device_parsing(
```

Comment:
Would maybe make more sense to call this `_parse_devices`

Reply:
Makes sense! Done